webtv/README.md

```diff
@@ -0,0 +1,96 @@
+# OpenPass Sample Demo App
+
+This is a [React.js](https://create-react-app.dev/) project that creates a generic app to be compiled to multiple WebTV Technologies.
+
+This app currently implements [OpenPass Javascript SDK](https://github.com/openpass-sso/openpass-js-sdk) to do the authorization flow for the app.
+
+Current WebTV technologies supported:
+
+- [WebOS](https://www.webosose.org/docs/home/)
+- [Tizen](https://developer.tizen.org/)
+
+## Recommended Versions
+
+- Use Node.js 16
+
+## Getting Started
```

Comment:
Might need the npm install command first

Reply:
Stepp added to the docs